maestro-ios-xctest-runner/maestro-driver-iosUITests/maestro_driver_iosUITests.swift

```diff
@@ -2,20 +2,29 @@ import XCTest
 import FlyingFox
 
 class maestro_driver_iosUITests: XCTestCase {
+    private static var swizzledOutIdle = false
 
     override func setUpWithError() throws {
-        // Put setup code here. This method is called before the invocation of each test method in the class.
+        if !maestro_driver_iosUITests.swizzledOutIdle { // ensure the swizzle only happens once
+            let original = class_getInstanceMethod(objc_getClass("XCUIApplicationProcess") as? AnyClass, Selector(("waitForQuiescenceIncludingAnimationsIdle:")))
+            let replaced = class_getInstanceMethod(type(of: self), #selector(maestro_driver_iosUITests.replace))
+            
+            guard let original = original, let replaced = replaced else { return }
+            
+            method_exchangeImplementations(original, replaced)
+            maestro_driver_iosUITests.swizzledOutIdle = true
+        }
 
         // In UI tests it is usually best to stop immediately when a failure occurs.
         continueAfterFailure = false
 
         // In UI tests it’s important to set the initial state - such as interface orientation - required for your tests before they run. The setUp method is a good place to do this.
     }
-
-    override func tearDownWithError() throws {
-        // Put teardown code here. This method is called after the invocation of each test method in the class.
+    
+    @objc func replace() {
```

Comment:
Lets call this `replace_waitForQuiescenceIncludingAnimationsIdle`, to make the swizzle a bit less confusing.

Reply:
And inside a comment:

```
    // Disable waiting for quiescence
```